packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRandom } from "@fluid-internal/stochastic-test-utils";
+import { DataObject } from "@fluidframework/aqueduct";
+import { assert } from "@fluidframework/common-utils";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedCounter } from "@fluidframework/counter";
+
+/**
+ * DataObjectWithCounter increments a SharedCounter as a way of sending ops.
+ *
+ * The SharedCounter is retrieved via handle
+ */
+const counterKey = "counter";
+export class DataObjectWithCounter extends DataObject {
+    private _counterHandle?: IFluidHandle<SharedCounter>;
```

Comment:
It is probably better to directly store the counter and expose that. Its cheaper to get the handle from the counter than vice-versa.